src/backend/Dockerfile.efiling-api

```diff
@@ -11,11 +11,12 @@ ARG STARTERS_V
 ENV SERVICE_NAME=${SERVICE_NAME}
 ENV STARTERS_V=${STARTERS_V}
 
-RUN mkdir /root/.ssh && \
-    chmod 0700 /root/.ssh  && \
-    ssh-keyscan -t rsa localhost >> ~/.ssh/known_hosts  && \
-    apt-get update  && \
-    apt-get -y install git
+#Temporary removal.
```

Comment:
Temporary removal ?

Reply:
I think it may be required. It may be required for sftp access in openshift.